test/public.spec.js

```diff
@@ -91,7 +94,7 @@ describe('imageChecker', () => {
       window.NCC.imageChecker.showImagesInfo();
 
       let imageOverlays = document.querySelectorAll('.ncc-image-checker-overlay');
-      expect(imageOverlays.length).toEqual(5);
+      expect(imageOverlays.length).toEqual(6);
```

Comment:
@tabanliviu INFO - So which condition added the extra overlay... `(empty)` or `{ poster: false }`?